[docs] Update CDC connector docs with poll interval changes and deployment warning (#30300)
yugabyte/yugabyte-db: docs/content/stable/additional-features/change-data-capture/using-logical-replication/yugabytedb-connector.md

3. Add the directory with the JAR files to the [Kafka Connect `plugin.path`](https://kafka.apache.org/documentation/#connectconfigs).
4. Restart your Kafka Connect process to pick up the new JAR files.

{{< warning title="Warning" >}}

> On connector version dz.2.5.2.yb.2025.2, users may face the following error while deploying the connector:

Reviewer: ```suggestion
Using connector version dz.2.5.2.yb.2025.2, you may get the following error while deploying the connector:
```
